.circleci/config.yml

```diff
@@ -672,7 +672,7 @@ workflows:
 
       - test-contracts:
           name: "test-perps-market"
-          requires: [build-testable]
+          requires: [test-spot-market, build-testable]
```

Comment:
There should not be dependency between tests, it is not a local monoreo and all tests are isolated. This dependency only increases build time without any benefits. Please remove this.